clients/lsp-php.el

```diff
@@ -33,16 +33,28 @@
   :link '(url-link "https://github.com/felixfbecker/php-language-server")
   :group 'lsp-mode)
 
-(defcustom lsp-clients-php-server-command
-  `("php" ,(expand-file-name "~/.composer/vendor/felixfbecker/language-server/bin/php-language-server.php"))
+(defcustom lsp-php-composer-dir
+  (if (executable-find "composer")
+      (replace-regexp-in-string "\n$" "" (shell-command-to-string "composer config --global home"))
+    "~/.composer")
+  "Home directory of composer."
+  :group 'lsp-php
+  :type 'string)
```

Comment:
this should be deleted and the calculation of the dir should happen in the lambda bellow.

Reply:
This variable (lsp-php-composer-dir) is used twice (one in lsp-clients-php-server-command, other in lsp-phpactor-path)
I moved it into defun to reuse it, is it ok?
Please help me check the latest commit, thanks for your support.